tamr_toolbox/data_io/df_connect/client.py

```diff
@@ -317,6 +326,44 @@ def export_dataset(
     return r.json()
 
 
+def export_delta_dataset(
```

Comment:
Rather than creating an extra function here, why don't we try adding a new parameter to the existing `export_dataset` function?  It could be `delta_versions: Optional[Tuple[str, str]] = None`, where passing `delta_versions=("1", "2")` is how you get the delta from version 1 to version 2.

It can then have a warning if `delta_versions` is not `None` and `truncate_before_load` is `True`, saying that delta exports cannot truncate the target table.